src/app/core/confirmation-dialog/confirmation-dialog.service.ts

```diff
@@ -55,4 +55,11 @@ export class ConfirmationDialogService {
         .afterClosed()
     );
   }
+
+  getSaveConfirmation() {
+    return this.getConfirmation(
+      $localize`:Save changes header:Save Changes?`,
```

Comment:
:warning: 
This is very misleading! Actually "Yes" is discarding changes, not saving them.

```suggestion
      $localize`:Save changes header:Discard Changes?`,
```